src/main/java/me/botsko/prism/serializers/entity/VillagerSerializer.java

```diff
@@ -0,0 +1,74 @@
+package me.botsko.prism.serializers.entity;
+
+import me.botsko.prism.serializers.items.ItemStackSerializer;
+import me.botsko.prism.utils.MiscUtils;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Villager;
+import org.bukkit.entity.Villager.Profession;
+import org.bukkit.inventory.MerchantRecipe;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+public class VillagerSerializer extends AbstractVillagerSerializer {
+
+    protected String type = null;
+    protected String profession = null;
+    protected Integer villagerXp = 0;
+    protected List<VillagerRecipe> recipes = new ArrayList<>();
+
+    @Override
+    protected void serializer(Entity entity) {
+        super.serializer(entity);
+        type = ((Villager) entity).getVillagerType().name().toLowerCase();
+        profession = ((Villager) entity).getProfession().name().toLowerCase();
+        villagerXp = ((Villager) entity).getVillagerExperience();
+        for (MerchantRecipe recipe : ((Villager) entity).getRecipes()) {
+            final VillagerRecipe r = new VillagerRecipe();
+            r.result = ItemStackSerializer.createItemStackSerialized(recipe.getResult());
+            r.ingredient = new ArrayList<>();
+            recipe.getIngredients().forEach(rec -> r.ingredient.add(ItemStackSerializer.createItemStackSerialized(rec)));
```

Comment:
Codacy found an issue: [Line is longer than 120 characters (found 121).](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=6368169)